dipy/reconst/mcsd.py

```diff
@@ -739,6 +739,8 @@ def auto_response_msmt(gtab, data, tol=20, roi_center=None, roi_radii=10,
     gtab : GradientTable
     data : ndarray
         diffusion data
+    tol : int
```

Comment:
`int, optional` instead

You can also update the other parameters of this function and add `, optional` then, we can merge.

Thanks!

Reply:
Done in 4d97d58.